lib/request-options.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+
+module.exports = (libOptions, logger) => {
+    /**
+     * @typedef {Object} options
+     * @property {Boolean} json Sets the body of the request to JSON.
+     * @proptery {Boolean} resolveWithFullResponse Get the full response
+     *                                             instead of just
+     *                                             the body.
+     * @property {Boolean} simple Get a rejection only if the request
+     *                            failed for technical reasons.
+     */
+
+    /**
+     * Preps options object for a Shelf request.
+     *
+     * @param {string} authToken The Shelf access token.
+     * @return {options}
+     */
+    function createOptions(authToken) {
+        var options;
+
+        logger.debug("In requestOptions.createOptions()");
+
+        options = {
+            headers: {
+                Authorization: authToken
+            },
+            json: true,
```

Comment:
Setting the `json` option to true attempts to stringify everything, including the headers (which is the main reason why I want it for everything). I don't think it hurts anything, and I would be setting it to true in every function except maybe `Artifact.download()` (to capture a non-string `response.body`).